### PR TITLE
Rb starting on UI

### PR DESCRIPTION
**Pull request by Cindellai**

### CODE REVIEW

File: app/controllers/application_controller.rb
Comment (jrzricardo) [inline app/controllers/application_controller.rb:1]: nice, gotta check these dudes first.

File: app/views/comments/index.html.erb
Comment (jrzricardo) [inline app/views/comments/index.html.erb:1]: yeh, get it outta here.

File: app/views/comments/show.html.erb
Comment (jrzricardo) [inline app/views/comments/show.html.erb:1]: OUTTA HERE.

File: app/views/follow_requests/index.html.erb
Comment (jrzricardo) [inline app/views/follow_requests/index.html.erb:1]: peace out green.

File: app/views/follow_requests/show.html.erb
Comment (jrzricardo) [inline app/views/follow_requests/show.html.erb:1]: toodles verde.

File: app/views/layouts/application.html.erb
Comment (jrzricardo) [inline app/views/layouts/application.html.erb:1]: i don't think including "partial" in the render is necessary, but I respect your choice to include it.

File: app/views/likes/index.html.erb
Comment (jrzricardo) [inline app/views/likes/index.html.erb:1]: get gone green.

File: app/views/likes/show.html.erb
Comment (jrzricardo) [inline app/views/likes/show.html.erb:1]: deuces green.

File: app/views/photos/index.html.erb
Comment (jrzricardo) [inline app/views/photos/index.html.erb:1]: who keeps letting green in?

File: app/views/photos/show.html.erb
Comment (jrzricardo) [inline app/views/photos/show.html.erb:1]: bye Felicia.

File: app/views/shared/_cdn_assets.html.erb
Comment (jrzricardo) [inline app/views/shared/_cdn_assets.html.erb:1]: this is the best copt/pasta I have ever seen.

File: app/views/shared/_flash.html.erb
Comment (jrzricardo) [inline app/views/shared/_flash.html.erb:1]: nice flash.

File: app/views/shared/_navbar.html.erb
Comment (jrzricardo) [inline app/views/shared/_navbar.html.erb:1]: we got bars now.

File: config/routes.rb
Comment (jrzricardo) [inline config/routes.rb:1]: I like that you went with the alphabetizing. looks nice :>

File: lib/tasks/dev.rake
Comment (jrzricardo) [inline lib/tasks/dev.rake:1]: alice & bob are here to stay.

**Review by jrzricardo — APPROVED** [review]
everything looks to be in good shape! nice going :>

File: app/views/shared/_navbar.html.erb [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+        </li>
+        <li class="nav-item">
+          <%= link_to "Link", "#", class: "nav-link" %>
+        </li>
+        <li class="nav-item dropdown">
+          <a class="nav-link dropdown-toggle" href="#" role="button" data-bs-toggle="dropdown" aria-expanded="false">
+            Resources
+          </a>
+          <div class="dropdown-menu">
+            <%= link_to "Photos", photos_path, class: "dropdown-item" %>
```
Comment (JoseMDO) [inline app/views/shared/_navbar.html.erb:20]: Nice job using helper methods!

File: lib/tasks/dev.rake [hunk 2]
```diff
@@ -9,6 +9,22 @@ task sample_data: :environment do
     User.destroy_all
   end
 
+  #knew one of our user accounts, so that we could log right in and keep testing the app
+  usernames = Array.new { Faker::Name.first_name }
+
+  usernames << "alice"
+  usernames << "bob"
+
+  usernames.each do |username|
```
Comment (JoseMDO) [inline lib/tasks/dev.rake:18]: Great job on your sample data!

File: app/views/layouts/application.html.erb [hunk 3]
```diff
@@ -5,12 +5,23 @@
     <meta name="viewport" content="width=device-width,initial-scale=1">
     <%= csrf_meta_tags %>
     <%= csp_meta_tag %>
-
+    <%= render partial: "shared/cdn_assets" %>
```
Comment (JoseMDO) [inline app/views/layouts/application.html.erb:8]: Nice job adding your CDN assets to a partial, this helps keep your layout file clean!

File: app/controllers/application_controller.rb [hunk 4]
```diff
@@ -1,2 +1,3 @@
 class ApplicationController < ActionController::Base
+  before_action :authenticate_user!
```
Comment (JoseMDO) [inline app/controllers/application_controller.rb:2]: Nice!

**Review by JoseMDO — APPROVED** [review]
Looks good!

**Pull request merged**